chainer/links/normalization/instance_normalization.py

```diff
@@ -0,0 +1,140 @@
+import chainer
+from chainer.functions.normalization import instance_normalization
+from chainer import initializers
+from chainer import link
+from chainer import variable
+
+
+class InstanceNormalization(link.Link):
+
+    """Instance normalization layer on outputs of linear or convolution functions.
+
+    This link wraps the :func:`~chainer.functions.instance_normalization`.
+    Instance normalization is very close to batch normalization but different
+    in that this normalizes each samples in mini batch by its mean and standard
```

Comment:
Grammar and minor cosmetic for consistency.
```suggestion
    in that this normalizes each samples in a mini-batch by its mean and standard
```